### ISSUE TITLE
HTTP Error 500: INTERNAL SERVER ERROR

### ISSUE THREAD

**Issue by rdemets** [kevinjohncutler/omnipose#7]
Hi, 

I have a conda environment that I used for cellpose 2.0, and I thought it would be quite straightforward to use the same environment for omnipose (I only did a `pip install omnipose` since). 

First, I am getting a HTTP Error 500: INTERNAL SERVER ERROR when I try to run the line
`model = models.Cellpose(gpu=use_GPU, model_type='bact_omni')` from the run_cellpose_GPU notebook. 

Secondly, if I try to run it using one model that I already downloaded, I am getting an error 
`eval() got an unexpected keyword argument 'omni'`
at the line
`masks, flows, styles, diams = model.eval(imgs_2D, diameter=None, flow_threshold=None, channels=channels, omni=True)`

Am I missing anything ? Is there any other way to download the model and drag and drop it with the other models ? Should I have a dedicated environment without cellpose ? It seems like an easy to fix problem, I am a bit clueless on the procedure to use Omnipose properly. 

Thanks for your help.
Regards, R.

**Comment by rdemets** [comment]
Nevermind, I found the problem. It's indeed the cellpose version. 
I uninstall cellpose and `pip install cellpose==1.0.2`

You can close this issue.
